src/Compilers/CSharp/Portable/Symbols/TypeSymbol.cs

```diff
@@ -517,23 +517,18 @@ internal Microsoft.Cci.PrimitiveTypeCode PrimitiveTypeCode
         #region Use-Site Diagnostics
 
         /// <summary>
-        /// Return error code that has highest priority while calculating use site error for this symbol. 
+        /// Returns true if the error code is highest priority while calculating use site error for this symbol. 
         /// </summary>
-        protected override int HighestPriorityUseSiteError
-        {
-            get
-            {
-                return (int)ErrorCode.ERR_BogusType;
-            }
-        }
+        protected sealed override bool IsHighestPriorityUseSiteErrorCode(int code)
+            => code is (int)ErrorCode.ERR_UnsupportedCompilerFeature or (int)ErrorCode.ERR_BindToBogus or (int)ErrorCode.ERR_BogusType;
```

Comment:
> or (int)ErrorCode.ERR_BogusType [](http://example.com/codeflow?start=102&length=32)

This doesn't look necessary.

Reply:
It looks like ```or (int)ErrorCode.ERR_BindToBogus``` doesn't belong in this check, we were checking only for ```ErrorCode.ERR_BogusType``` before.

Reply:
Tests fail without this change.

Reply:
Specifically, the `BogusType` part that you pointed out.